fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -2423,8 +2423,7 @@ public List<ResultRow> executeInternalQuery() {
                     analyze(context.getSessionVariable().toThrift());
                 }
             } catch (Exception e) {
-                LOG.warn("Internal SQL execution failed, SQL: {}", originStmt, e);
```

Comment:
why remove log?

Reply:
We would see the same message in the caller side since we throw an exception now